cloud/amazon/ec2_vol.py

```diff
@@ -332,7 +338,10 @@ def create_volume(module, ec2, zone):
     if volume is None:
         try:
             if boto_supports_volume_encryption():
-                volume = ec2.create_volume(volume_size, zone, snapshot, volume_type, iops, encrypted)
+                if kms_key_id:
```

Comment:
Can you add a check for boto version here? Something like 
Add a new import
`from distutils.version import StrictVersion`

```python
if StrictVersion(boto.__version__) < StrictVersion('2.39.0'):
    module.fail_json(msg="A newer version of boto is required, %s detected but 2.39.0 is required for kms_key_id" % boto.__version__)